MediaBrowser.Controller/Entities/Video.cs

```diff
@@ -456,7 +456,7 @@ private void RefreshLinkedAlternateVersions()
             foreach (var child in LinkedAlternateVersions)
             {
                 // Reset the cached value
-                if (child.ItemId.HasValue && child.ItemId.Value.Equals(default))
+                if (child.ItemId.IsNullOrEmpty())
```

Comment:
This changes behavior, the existing code checks that it is empty but _not_ `null`

Reply:
I don't think the difference is meaningful enough to keep the `.HasValue` check, now a Guid? that is already null can be set to null again

Reply:
Didn't see the body of the if, makes sense